CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # Changelog
 
+#### Added  2024-04-26
+
+* The `chaosgcp.iam.controls.policy` control to Manage temporary, time-bound IAM roles for the specified project and members.
+* This controls grants specified IAM roles to the provided members, with an expiration time determined by the given expiry_time_in_minutes  before experiment and revokes the specified IAM roles from the provided members after the experiment:
```

Comment:
Try to keep each column under a 80-width rule. I know we have bigger screens but I think it's still easier to read with narrower width.

Reply:
modified changelog with 80-width rule